internal/config/config.go

```diff
@@ -103,3 +122,13 @@ func (c *Config) AssignClients() map[string][]hotstuff.ID {
 	}
 	return hostsToClients
 }
+
+// Returns true if both the replica and client hosts slices
```

Comment:
It is common (unless recommendations have changed) to start doc comments with the name of the func.
```go
// IsLocal returns true ...
```